test/integration/ClientConstantsIntegrationTest.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable mocha/no-setup-in-describe */
+import {
+    MAINNET,
+    WEB_TESTNET,
+    WEB_PREVIEWNET,
+    NATIVE_PREVIEWNET,
+    NATIVE_TESTNET,
+} from "../../src/constants/ClientConstants.js";
+import {
+    AccountBalance,
+    AccountBalanceQuery,
+    Hbar,
+} from "../../src/exports.js";
+import IntegrationTestEnv from "./client/NodeIntegrationTestEnv.js";
+
+describe("ClientConstantsIntegrationTest", function () {
+    let env;
+
+    before(async function () {
+        env = await IntegrationTestEnv.new();
+    });
+
+    describe("MAINNET node proxies", function () {
+        const proxies = Object.keys(MAINNET);
+        proxies.forEach((proxy) => {
+            it(`should fetch ${MAINNET[proxy]} account balnace`, async function () {
+                const accountBalance = await new AccountBalanceQuery()
+                    .setNodeAccountIds([MAINNET[proxy]])
+                    .setAccountId(env.operatorId)
```

Comment:
I think we should use a known account like `0.0.3` for checking the balance as we might run into some problems if the operator account id doesn't exist on all networks